docs/docs/command-line-flags.md

```diff
@@ -125,7 +125,47 @@ The core functionality flags can be also set by environment variable `MARATHON_O
 Mesos frequently sends resource offers to Marathon (and all other frameworks). Each offer will represent the
 available resources of a single node in the cluster. Before this <span class="label label-default">v0.8.2</span>,
 Marathon would only start a single task per
-resource offer, which led to slow task launching in smaller clusters. In order to speed up task launching and use the
+resource offer, which led to slow task launching in smaller clusters. 
+
+### Marathon after 0.11.0 (including)
```

Comment:
On one hand is clear, that these properties are available with 0.11.0 
On the other hand it would be easier to have only one list to read ....

I am a bigger fan of have all docs versioned with a flat list.